engine/animation/staticSprite.hpp

```diff
@@ -22,7 +22,9 @@ class staticSprite : public AbstractSpriteHandler{
       dstrect.y = round((_pScene->mDimmension[1]-mPosition[1]-mDimmension[1]) * zoom);
       dstrect.w = round(zoom * mDimmension[0]);
       dstrect.h = round(zoom * mDimmension[1]);
-      SDL_RenderCopy(_renderer, mpSprite, nullptr, &dstrect);
+
+      cam->renderCopy(mpSprite, nullptr, &dstrect);
```

Comment:
Maybe there's a better name for it that renderCopy? Like displayTexture? The SDL functions use the Render prefix because the functions acts on an SDL_Renderer* but it's a bit unnecessary for us.